src/main.rs

```diff
@@ -162,6 +168,10 @@ fn main() {
     let args: Args = Docopt::new(USAGE)
                             .and_then(|d| d.deserialize())
                             .unwrap_or_else(|e| e.exit());
+    
+    if args.flag_pager {
+        Pager::new().setup();
```

Comment:
Maybe we should add the `.skip_on_notty()` option?

Reply:
I thought the same, but according to the rust compiler that options is enable by default.

```
warning: use of deprecated item 'pager::Pager::skip_on_notty': 'skip_on_notty' is default now
   --> src/main.rs:173:22
    |
173 |         Pager::new().skip_on_notty().setup();
    |                      ^^^^^^^^^^^^^
    |
    = note: #[warn(deprecated)] on by default

```

Reply:
You're right, then it's better to remove it again :)